### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by amjadiftikhar**

### CODE REVIEW

File: react/week1/todo-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+// import logo from "./logo.svg";
+import "./App.css";
+
+class App extends React.Component {
```
Comment (senner008) [inline react/week1/todo-app/src/App.js]: Hi:)
For week 2 homework I would like you to try and rewrite this app using function components with Hooks. Just as we did in class.
Let me know how it goes

**Comment by senner008** [comment]
Also avoid calling it 'Add files via upload' This is not descriptive

File: react/week1/todo-app/src/App.js (hunk 1 above)
Comment (amjadiftikhar) [reply react/week1/todo-app/src/App.js]: I have done the app with function components.

**Comment by senner008** [comment]
Ok 👍

**Pull request merged**